tests/python_client/testcases/test_collection.py

```diff
@@ -2725,3 +2710,117 @@ def test_load_release_after_collection_drop(self, connect, collection):
         error = {ct.err_code: 0, ct.err_msg: "HasPartition failed: can\'t find collection: %s" % name}
         partition_w.load(check_task=CheckTasks.err_res, check_items=error)
         partition_w.release(check_task=CheckTasks.err_res, check_items=error)
+
+
+class TestCollectionString(TestcaseBase):
+    """
+    ******************************************************************
+      The following cases are used to test about string 
+    ******************************************************************
+    """
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_collection_string_field_is_primary(self):
+        """
+        target: test create collection with string field
+        method: 1. create collection with string field and vector field
+                2. set string fields is_primary=True
+        expected: Create  collection Successfully
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        int_field = cf.gen_int64_field()
+        vec_field = cf.gen_float_vec_field()
+        string_field = cf.gen_string_field(is_primary=True)
+        schema = cf.gen_collection_schema(fields=[int_field, string_field, vec_field])
+        self.collection_wrap.init_collection(name=c_name, schema=schema, check_task=CheckTasks.check_collection_property,
+                                             check_items={exp_name: c_name, exp_schema: schema})
+    
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_collection_with_muti_string_fields(self):
+        """
+        target: test create collection with muti string fields
+        method: 1. create  collection with primary string field and not primary string field
+                2.  String fields is_primary=True
+        expected: Create  collection Successfully
```

Comment:
Create  collection Successfully -> Create collection Successfully (two blanks) and the above "string" is not aligned with "create ..." line